src/everest/config/install_data_config.py

```diff
@@ -40,7 +71,36 @@ def validate_link_type(cls, link: bool | None) -> bool | None:
         return link
 
     @model_validator(mode="after")
-    def validate_target(self) -> "InstallDataConfig":
-        if self.target in {".", "./"}:
+    def validate_target(self) -> InstallDataConfig:
+        if self.data is not None:
+            if self.source is not None:
+                raise ValueError("The data and source options are mutually exclusive.")
+            if self.target.strip() in {"", ".", "./"}:
+                raise ValueError("A target name must be provided with data.")
+
```

Comment:
Is it an idea to test that the file location is writable?

Reply:
The file location is the run path, so it does not exist yet. The target file will be written by the same copy forward model as used by `install_data`, it presumable uses the necessary safeguards.